src/tv2-common/helpers/graphics/pilot/index.ts

```diff
@@ -129,17 +136,17 @@ function CreatePilotAdLibAction(
 	})
 }
 
-export function CreateFullPiece(
-	config: TV2BlueprintConfig,
-	context: IShowStyleUserContext,
-	partId: string,
-	parsedCue: CueDefinitionGraphic<GraphicPilot>,
-	engine: GraphicEngine,
-	settings: PilotGeneratorSettings,
-	adlib: boolean,
-	segmentExternalId: string,
-	prerollDuration?: number
-): IBlueprintPiece {
+export function CreatePilotPiece({
```

Comment:
I don't think we have any other "pilots" anywhere, it's all about graphics made in Viz Content Pilot.

Reply:
An argument could be made that we should still call it `CreatePilotGraphicPiece` because it's going to be a real pain to refactor if we introduce another Pilot thing. But that's YAGNI, so lets burn that brigde when we get to it :)